### DISCUSSION TITLE
replit 限制部署如何解决

### DISCUSSION THREAD

**Discussion by HurricanMaple** [AlistGo/alist#3499] [Q&A] [answered]
各位大佬们请问 replit 禁止部署 alist 了你们是怎么解决这个问题的，之前部署的也不能用了，我尝试使用命令去安装，提示不可用然后我这个 repl 被封了不能使用

**Comment by HurricanMaple** [comment]
现在大概有解决办法了，下载包上传命令行解压，然后运行，数据库的话不知道如何解决，数据库不知道可不可以通过.replit 修改

**Comment by HurricanMaple** [comment]
![Screenshot_20230218_170557_com microsoft emmx](https://user-images.githubusercontent.com/112387102/219851921-e99773d5-59de-4c18-88e1-e5287e8f408f.jpg)
手动上传文件结果还是会被检测怎么办

**Comment by HurricanMaple** [comment, marked as answer]
终于搞好了，把文件改名否则很容易检测，然后更改配置文件去改数据库，然后运行启动命令就可以绕过检测了

**Comment by xx6z8v** [comment]
你好,我也遇到同样的问题,方便详细说说你的解决方案吗

**Reply by HurricanMaple** [reply]
下载官方文件，将官方文件修改名字，导入 replit，具体可以看我博客 blog.alcex.top

**Comment by PennHui666** [comment]
你好，请问这种方法部署的要如何更新？把新发布的包上传后再解压覆盖alist文件吗？

**Comment by StartMoney** [comment]
怎么改成mysql远程服务器？怕资源泄露
